additionaldocs/sagemaker/install-graph-explorer-lc.sh

```diff
@@ -2,26 +2,94 @@
 
 sudo -u ec2-user -i <<'EOF'
 
-echo "export GRAPH_NOTEBOOK_AUTH_MODE=DEFAULT" >> ~/.bashrc  # set to IAM instead of DEFAULT if cluster is IAM enabled
-echo "export GRAPH_NOTEBOOK_HOST=CHANGE-ME" >> ~/.bashrc
-echo "export GRAPH_NOTEBOOK_SERVICE=neptune-db" >> ~/.bashrc # set to `neptune-db` for Neptune database or `neptune-graph` for Neptune Analytics
-echo "export GRAPH_NOTEBOOK_PORT=8182" >> ~/.bashrc
-echo "export AWS_REGION=us-west-2" >> ~/.bashrc  # modify region if needed
+echo "export GRAPH_NOTEBOOK_SERVICE=neptune-db" >> ~/.bashrc # Set to `neptune-graph` for Neptune Analytics.
+echo "export GRAPH_NOTEBOOK_HOST=CHANGE-ME" >> ~/.bashrc # Set to the endpoint of your Neptune instance
+echo "export GRAPH_NOTEBOOK_PORT=8182" >> ~/.bashrc # Set to the port of your Neptune instance
+echo "export GRAPH_NOTEBOOK_AUTH_MODE=IAM" >> ~/.bashrc # Set to `DEFAULT` if using DB cluster that is not IAM enabled. Leave as `IAM` for Neptune Analytics.
```

Comment:
@michaelnchin Should we make the default value be `DEFAULT` instead of `IAM`?

Reply:
`IAM` is likely to be applicable to more cases since it's required for Neptune Analytics, but can be either for Neptune Database. So IMO we should leave it as is